prime-router/settings/organizations-local.yml

```diff
@@ -566,7 +566,7 @@
           initialTime: 00:00
         translation:
           type: CUSTOM
-          schemaName: tx/tx-covid-19
```

Comment:
I decided to have the new SFTP_LEGACY use TX, so old HL7 I switched to using FL schema.
I like using "real" state schemas, even though the data is not actually going to those states, because it makes a more realistic end-to-end style test.